src/samples/audio/kwd_nn_detect_test.c

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2021 NXP
+//
+// Author: Cristina Feies <cristina.ilie@nxp.com>
+// Author: Viorel Suman <viorel.suman@nxp.com>
+
+#include <sof/lib/wait.h>
+#include <sof/samples/audio/kwd_nn/kwd_nn_process.h>
+#include <sof/samples/audio/kwd_nn/kwd_nn_preprocess.h>
+#include <sof/audio/component.h>
+#include <sof/samples/audio/detect_test.h>
+#include <sof/samples/audio/kwd_nn_detect_test.h>
+#include <stdint.h>
```

Comment:
Most specific to most generic. So first should be the kwd_nn ones (in alphabetic order), followed by the other SOF ones (such as <sof/audio/component.h>), followed by the libc ones (<stdint.h>).